binding/Binding/SKData.cs

```diff
@@ -251,6 +251,7 @@ public void SaveTo (Stream target)
 			} finally {
 				ArrayPool<byte>.Shared.Return (buffer);
 			}
+			GC.KeepAlive (this);
```

Comment:
Just prevent the GC from going to town, because we are not actually using the object, but a value from one of the properties.